src/app/modules/common/console-footer/console-footer.component.scss

```diff
@@ -0,0 +1,23 @@
+.footer-console-bar {
+  background-color: #000;
```

Comment:
Should'n we use css VARiables?

Reply:
Console colors stay the same regardless of the theme selected.